[2022-09-18 03:00:35,358] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: first_py_dag.python_task_2 2022-09-17T03:00:00+00:00 [queued]>
[2022-09-18 03:00:35,482] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: first_py_dag.python_task_2 2022-09-17T03:00:00+00:00 [queued]>
[2022-09-18 03:00:35,484] {taskinstance.py:1017} INFO - 
--------------------------------------------------------------------------------
[2022-09-18 03:00:35,494] {taskinstance.py:1018} INFO - Starting attempt 1 of 3
[2022-09-18 03:00:35,496] {taskinstance.py:1019} INFO - 
--------------------------------------------------------------------------------
[2022-09-18 03:00:35,680] {taskinstance.py:1038} INFO - Executing <Task(PythonOperator): python_task_2> on 2022-09-17T03:00:00+00:00
[2022-09-18 03:00:35,820] {standard_task_runner.py:51} INFO - Started process 26702 to run task
[2022-09-18 03:00:35,891] {standard_task_runner.py:75} INFO - Running: ['airflow', 'tasks', 'run', 'first_py_dag', 'python_task_2', '2022-09-17T03:00:00+00:00', '--job-id', '243', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_python_operator.py', '--cfg-path', '/tmp/tmpsgza40au']
[2022-09-18 03:00:35,915] {standard_task_runner.py:76} INFO - Job 243: Subtask python_task_2
[2022-09-18 03:00:36,436] {logging_mixin.py:103} INFO - Running <TaskInstance: first_py_dag.python_task_2 2022-09-17T03:00:00+00:00 [running]> on host 27377494ab8b
[2022-09-18 03:00:36,984] {taskinstance.py:1232} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Dave
AIRFLOW_CTX_DAG_ID=first_py_dag
AIRFLOW_CTX_TASK_ID=python_task_2
AIRFLOW_CTX_EXECUTION_DATE=2022-09-17T03:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-09-17T03:00:00+00:00
[2022-09-18 03:00:37,147] {python.py:118} INFO - Done. Returned value was: None
[2022-09-18 03:00:37,257] {taskinstance.py:1142} INFO - Marking task as SUCCESS. dag_id=first_py_dag, task_id=python_task_2, execution_date=20220917T030000, start_date=20220918T030035, end_date=20220918T030037
[2022-09-18 03:00:41,853] {taskinstance.py:1195} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-09-18 03:00:42,217] {local_task_job.py:170} WARNING - State of this instance has been externally set to success. Terminating instance.
[2022-09-18 03:00:43,355] {process_utils.py:95} INFO - Sending Signals.SIGTERM to GPID 26702
[2022-09-18 03:00:43,508] {process_utils.py:61} INFO - Process psutil.Process(pid=26702, status='terminated', exitcode=0, started='03:00:35') (26702) terminated with exit code 0
[2022-09-18 03:00:43,528] {local_task_job.py:118} INFO - Task exited with return code 0
